src/FixedDataTableColumn.js

```diff
@@ -149,9 +149,9 @@ class FixedDataTableColumn extends React.Component {
    flexGrow: PropTypes.number,
 
    /**
-    * Whether the column can be resized with the
-    * FixedDataTableColumnResizeHandle. Please note that if a column
-    * has a flex grow, once you resize the column this will be set to 0.
+    * Whether the column can be resized with the ColumnResizerLine.
```

Comment:
Should this actually be resize handle in FixedDataTableCell?

Reply:
@wcjordan, hmm, technically you're right. It's the resize handle that initiates the ColumnResizerLine.
Do you think I should just refer to "the resize handle" here? (There's no react class here)

Reply:
Ya, I would just say whether the column can be resized.

Reply:
sure, will drop it.